rust/threshold-mock/README.md

```diff
@@ -0,0 +1,31 @@
+# Chain-key testing canister
+
+The chain-key testing canister is a canister smart contract that provides a [fake](https://www.martinfowler.com/articles/mocksArentStubs.html#TheDifferenceBetweenMocksAndStubs) implementation of the APIs of various chain-key-related features including threshold ECDSA, threshold Schnorr, and threshold key derivation (vetKeys) **for testing purposes**. The shortcut that this canister takes (compared to the production APIs) is that it relies on a cryptographic key that is hard-coded into the canister, rather than relying on a key that is distributed among the subnet nodes. With this, the implementation is inherently **insecure**: see [Disclaimer](#disclaimer).
```

Comment:
So in the link, there are two separate notions: a fake and a mock. We are using both here :)

Reply:
Do you still see "mock" somewhere? I though I removed all places where "mock" was used in the README and across the repository.

Reply:
But the new repository name doesn't have `mock` in the name, so all good